src/clj/game/cards/agendas.clj

```diff
@@ -204,6 +204,19 @@
                  :effect arrange-rd}
         :interactive (req true)}))
 
+   "Better Citizen Program"
+   {:events {:play-event {:req (req (has-subtype? target "Run"))
+                          :delayed-completion true
+                          :msg "give the Runner a tag for playing a run event"
+                          :effect (effect (tag-runner :runner eid 1))
+                          :once :per-turn}
+             :runner-install {:silent (req true)
+                              :req (req (has-subtype? target "Icebreaker"))
+                              :delayed-completion true
+                              :msg "give the Runner a tag for installing an icebreaker"
+                              :effect (effect (tag-runner :runner eid 1))
+                              :once :per-turn}}}
```

Comment:
This should technically use `first-event` instead of `:once`, as it's possible that a runner could have already installed an icebreaker on the turn the corp scores BCP, then installs another during the end-of-turn window (which should not trigger BCP). The same would apply to `:play-event` if the Runner could play events on corp's turn, but alas...

Reply:
swapped `{:events {:play-event {:req (req (has-subtype? target "Run"))`

with `{:events {:play-event {:req (req (first-event? state :runner :run))`